challenge-16/challenge-16.js

```diff
@@ -29,7 +34,9 @@ curso para fazer isso funcionar corretamente :)
 console.log para cada formato.
 */
 console.log( '\nNome convertido à partir de um slug:' );
-// ?
+var fullName = 'mateus-jabour';
+var newFullName = fullName.replace(fullName[0], fullName[0].toUpperCase()).replace(fullName[fullName.indexOf('-') + 1], fullName[fullName.indexOf('-') + 1].toUpperCase()).split('-').join(' ');
```

Comment:
-1
Sempre que for repetir código, atribua o que será repetido à uma variável. Deixa o código mais legível e fácil de dar manutenção :)